tests/db_functions/models.py

```diff
@@ -41,3 +41,12 @@ class Fan(models.Model):
 
     def __str__(self):
         return self.name
+
+
+@python_2_unicode_compatible
+class Number(models.Model):
+    integer = models.BigIntegerField()
+    float = models.FloatField(null=True)
```

Comment:
Apart from being used in `__str__` and `setUpTestData` the `float` field seems unused. We should either add an extra test to cast it somehow or simply remove it.